e2e_test/udf/test.py

```diff
@@ -84,7 +90,7 @@ def hex_to_dec(hex: Optional[str]) -> Optional[Decimal]:
     return dec
 
 
-@udf(input_types=["FLOAT8"], result_type="DECIMAL")
+@udf(input_types=["FLOAT64"], result_type="DECIMAL")
```

Comment:
Is this considered a breaking change that should be notable for existing users?

Reply:
Yes, it is a breaking change, but only for the new version. Existing users will not be affected unless they try to upgrade to the new version.